Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.14.10 AS build-env
+
+RUN rm -rf /etc/localtime \
+    && ln -s /usr/share/zoneinfo/Hongkong /etc/localtime \
```

Comment:
why we use hongkong timezone here..

Reply:
All dockerfiles in our products use this time zone, we can also set it to Asia/Shanghai.